jenkins_pipelines/scripts/json_generator/maintenance_json_generator.py

```diff
@@ -63,6 +63,7 @@
     "alma8_minion": {"/SUSE_Updates_RES_8-CLIENT-TOOLS_x86_64/"},
     "ubuntu2004_minion": {"/SUSE_Updates_Ubuntu_20.04-CLIENT-TOOLS_x86_64/"},
     "ubuntu2204_minion": {"/SUSE_Updates_Ubuntu_22.04-CLIENT-TOOLS_x86_64/"},
+    # no Ubuntu 24.04 on 4.3
```

Comment:
we need this for 5.0 as well though, the dictionaries are merged. Or add it in the 5.0 with the non beta one.

Reply:
i added to the 5.0 section.

For the moment the tools are not published to SCC, maybe my addition to 5.0 section was wrong?